pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1520,6 +1529,10 @@ class Timedelta(_Timedelta):
         div = other // self
         return div, other - div * self
 
+    # GH40946
```

Comment:
doc string and types pls

Reply:
Done on premise. (Depends on the decision to separate the issues in two PRs or not)